### PR TITLE
Addon Fixes

### PR DESCRIPTION
**Pull request by TheColaber**
**Changes**

I've stopped errors from occurring if using the `Last edit tooltip` addon when the project is unshared. I've also added a warning to the `Customizable block colors` addon because it uses traps.

**Reason for changes**

Just because

**Tests**

Works on chrome

Edit: I don't know how the first commit got there...
Edit: Resolves: #583

### CODE REVIEW

**Comment by WorldLanguages** [comment]
Customizable block colors does not use traps.

**Comment by ZenithRogue** [comment]
Why did you add that warning to Theme3? It doesn't impact performance at all, it is just a css change....

**Comment by TheColaber** [comment]
> Customizable block colors does not use traps.

It does, it is in the addon.json

> Why did you add that warning to Theme3? It doesn't impact performance at all, it is just a css change....

I tried it one Getting Over It on scratch and when I had only that addon enabled, it was slowing the performance... try yourself...

**Comment by ZenithRogue** [comment]
> > Customizable block colors does not use traps.
> 
> It does, it is in the addon.json

What? Show me

**Comment by TheColaber** [comment]
https://github.com/ScratchAddons/ScratchAddons/blob/master/addons/editor-theme3/addon.json#L89

**Comment by TheColaber** [comment]
Or I guess that line should be removed since you don't actually use any...

File: addons/exact-count/user.js [hunk 1]
```diff
@@ -3,47 +3,68 @@ export default async function ({ addon, global, console }) {
   let user1 = url.substring(30, 100);
   let username = user1.substring(0, user1.indexOf("/"));
   let dalength = document.getElementsByClassName("box-head").length;
+  var boxNumber = 1;
+  getUserData(
+    {
+      boxheadName: "Shared Projects",
+      url: "projects",
+    },
+    function () {
```
Comment (WorldLanguages) [inline addons/exact-count/user.js:12]: Why callbacks?? Just make it a function below the other, please

**Comment by WorldLanguages** [comment]
For the version of exact count on this PR: I've tried using it with my Scratch language set to Spanish, and it only set the number on shared projects, but not any other row. Can you fix that?

File: addons/exact-count/user.js (hunk 1 above)
Comment (TheColaber) [reply addons/exact-count/user.js:12]: I tried, but the API gives the data at random times and it messed up everything so I was trying to give it an order... is there a different solution?

**Comment by TheColaber** [comment]
> For the version of exact count on this PR: I've tried using it with my Scratch language set to Spanish, and it only set the number on shared projects, but not any other row. Can you fix that?

I'll see what I can do.

**Comment by TheColaber** [comment]
> For the version of exact count on this PR: I've tried using it with my Scratch language set to Spanish, and it only set the number on shared projects, but not any other row. Can you fix that?

Oh, I had it right on my end... I just forgot to update it. It was updated on #628

File: addons/exact-count/user.js (hunk 1 above)
Comment (WorldLanguages) [reply addons/exact-count/user.js:11]: You should pass `boxNumber` to each function then

Comment (TheColaber) [reply addons/exact-count/user.js:11]: But what if that person has not joined any studio or followed any? Then those boxes would disappear and the count would be wrong.

**Comment by TheColaber** [comment]
Wait why?

File: addons/exact-count/user.js (hunk 1 above)
Comment (WorldLanguages) [reply addons/exact-count/user.js:11]: You should first check what rows exist - then, request the counts.

**Comment by TheColaber** [comment]
@WorldLanguages Review?

**Pull request merged**